test/specs/vue.ts

```diff
@@ -1,9 +1,12 @@
 import assert = require('power-assert')
 import sinon = require('sinon')
-import Vue = require('vue')
+import Vue from 'vue'
 import { module, store, Getters, Mutations } from '../../src'
 
 describe('Vue integration', () => {
+  Vue.config.errorHandler = (err, vm, info) => {
```

Comment:
Yes, it would be better in `setup.ts` 🙂 
btw, can we leave a comment for describing the intention?